Doc/library/logging.rst

```diff
@@ -686,6 +687,12 @@ which has a ``filter`` method with the same semantics.
    parameter. The returned value should conform to that returned by
    :meth:`~Filter.filter`.
 
+.. versionchanged:: 3.12
+   You can now return a :class:`LogRecord` instance from filters to replace
+   the log record without modifying it in place. This allows filters attached to
+   on a :class:`Handler` to modify the log record before it is emitted without
```

Comment:
_attached to a handler_, not _attached to on a handler_.

Reply:
thank you, corrected